docs/order_of_PCI_addresses.md

```diff
@@ -0,0 +1,28 @@
+Order of PCI addresses
+===============================
+
+## Standard PCI addressing
+
+Coreboots standard way of numbering PCI devices prioritizes PCIe slots over
+embedded PCI devices like Ethernet interfaces. It causes weird and unwanted
```

Comment:
This is not exactly true. coreboot doesn't do anything wrong. It was the developer who initially wrote the support for apu2 and defined the PCIe engines in weird order to be initialized by AGESA. What coreboot does it to enumerate the PCI devices in ascending order as it should. I would remove this section and write it from scratch.

Write about:
- bad PCIe engines definition passed to AGESA in the initial coreboot port  for apu2 resulting in impractical order of PCIe devices